### PR TITLE
Add Link Quality alarm value to the OSD tab

### PR DESCRIPTION
**Pull request by McGiverGim**
Now that ELRS is the main radio control link, I think is important to let the user config the link quality (the recommended rssi value to be used with ELRS) directly in the Configurator OSD tab, like other alarms.

image

This PR needs to be merged after the firmware part: https://github.com/betaflight/betaflight/pull/13121

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Review by asizon — APPROVED** [review]

**Comment by nerdCopter** [comment]
https://github.com/betaflight/betaflight/pull/13121 needs a 3rd approval.

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
